modules/ROOT/pages/fcct-config.adoc

```diff
@@ -0,0 +1,107 @@
+:experimental:
+= FCCT Configuration and Common Use Cases
+
+== Scope
+This document provides the configuration specification for human-readable FCCT YAML files. Once written, use fcct to convert the YAML file into the machine-readable JSON Ignition file.
+
+Fedora CoreOS (FCOS) Ignition files specify the configuration of FCOS. The process begins with a YAML configuration file. The FCOS Configuration Transpiler (FCCT) converts the human-readable YAML file into machine-readable JSON.
+
+== FCCT Configuration Specification
+
+Refer to the https://github.com/coreos/fcct/blob/master/docs/configuration-v1_0.md[Configuration Specification] for a full list of options.
+
+== Common Use Cases
+
+=== Running Containers
+
+Fedora CoreOS ships with both docker and podman installed. Use systemd units to start and stop containers.
+
+.Example for running busybox using systemd and podman
```

Comment:
I'd prefer keeping this on its own page which shows "how to run a system service container".

For simpler examples, let's stick to "use this SSH pubkey" or "write this simple file to /etc/foo".